fdbclient/include/fdbclient/FDBTypes.h

```diff
@@ -1402,6 +1402,22 @@ struct TenantMode {
 		serializer(ar, mode);
 	}
 
+	// This does not go back-and-forth cleanly with toString
+	// The '_experimental' suffix, if present, needs to be removed in order to be parsed.
+	static TenantMode fromString(std::string mode) {
```

Comment:
@ajbeamon I also had the idea of processing the string here, so this helper could accept things like both `optional` and `optional_experimental` (which is what `toString()` provides). Would that be preferable over pre-processing the names in other places?